sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -58,8 +58,7 @@
 ]
 
 
-def raise_with_traceback(exception, *args, **kwargs):
-    # type: (Callable, Any, Any) -> None
+def raise_with_traceback(exception: Callable, *args, **kwargs) -> None:
```

Comment:
We should open an issue to re-review this in light of PEP3134

Reply:
@johanste what you had in mind? https://github.com/Azure/azure-sdk-for-python/issues/26760

Reply:
Yeah, that is what I was thinking. I didn't remember if there was an obscure scenario (apart from backwards combat) where we'd need to use the method.